user/syscalls.c

```diff
@@ -56,9 +100,22 @@ ssize_t write(int fd, const void *buf, size_t count) {
   return retval;
 }
 
-_off_t lseek(int __fildes, _off_t __offset, int __whence) {
-  errno = ENOSYS;
-  return (off_t)-1;
+_off_t lseek(int fd, _off_t offset, int whence) {
```

Comment:
I made a commit in an experimental branch which has a working implementation: 4b210edc44e8bcd96f6c9973ea11cd608aef0480. It's not as bad as I expected it to be. I'll merge it into this branch if you want.